style.css

```diff
@@ -0,0 +1,109 @@
+/*---------CSS reset---------------------------*/
+*{
+    margin: 0;
+}
+@font-face{
+    font-family: fixedsys;
```

Comment:
Font-family can store multiple font values which is great incase our browser doesn't support our first choice. We can ensure the 2nd/3rd/4th etc choice would still look good in our design.